plugins/badges-backend/README.md

```diff
@@ -0,0 +1,32 @@
+# Badges Backend
+
+Backend plugin for serving badges. Default implementation uses
+[badge-maker](https://www.npmjs.com/package/badge-maker) for creating
+the badges, in svg.
+
+## Setup
+
+Define which badges to offer in the backend api by declaring them in
+the app-config, under a `badges` key. Example:
+
+```yaml
+badges:
+  docs:
+    kind: 'entity'
+    target: '_{entity_url}/docs'
```

Comment:
Would add a description to repeat what you said below like "Entity documentation badge"

Reply:
Not sure I follow what you suggest here..